requirements.bazel.txt

```diff
@@ -26,8 +26,8 @@ pyasn1==0.5.0
 rsa==4.9
 greenlet==1.1.3.post0
 zope.interface==6.1
-opentelemetry-sdk==1.21.0
-opentelemetry-api==1.21.0
```

Comment:
Let's keep the pin for Bazel unless we have a strong reason not to. We want the testing to be deterministic.